ManageIQ/manageiq
# [POC] Use ActiveSupport::Logger.broadcast instead of MulticastLogger

Makes a switch to use `ActiveSupport::Logger.broadcast` which is already used to by the `rails console` to handle printing to both `STDOUT` and the log file for the rails `development.log`.  This just makes use of the same functionality that is there that is pretty much being replicated in the `MulticastLogger` that we put together ourselves.

Minor difference that shouldn't matter:  We now only control the level of the main logger, and that passes it to other loggers via super.  That really didn't matter since we don't multicast to more than two loggers, and the top one is the one is the only one that matters.  The `$container_log` will always be debug anyway, so no need for the level wrapper method that we had in `MulticastLogger`.


Links
-----
* Based off of https://github.com/ManageIQ/manageiq/pull/17223
* Diff between this branch and the above:  [diff](https://github.com/NickLaMuro/manageiq/compare/fix_datetime_format_method_for_multicast_logger...swap_out_multicast_logger_for_active_support_logger_broadcast)


Steps for Testing/QA
--------------------

Right now, just using tests for QA, but we might want to spin up some containers and see if things are still working as expected.
